Documentation/QueryDemo/QueryDemo.xcodeproj/project.pbxproj

```diff
@@ -78,7 +78,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				56519DC0274FC3F700ED16D8 /* GRDB in Frameworks */,
+				69BAF48A27AAB84800452053 /* GRDB in Frameworks */,
```

Comment:
This are some weird things Xcode has done, it was stuck on package resolving... now it uses the latest GRDB version, I can revert this changes if you want though